src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -1021,6 +1049,30 @@ public function iShouldBeAbleToGoToTheShippingStepAgain()
     }
 
     /**
+     * @Then I should be able to go to the complete step again
+     */
+    public function iShouldBeAbleToGoToTheCompleteStepAgain()
+    {
+        Assert::true(
+            $this->completePage->isOpen(),
```

Comment:
Yea, but in previous steps there is clicked button `next step`, so we are already on completePage. 
`Then I should be redirected to the shipping step`
 `And I should not be able to go to the payment step`
  `And I should be able to go to the complete step again`

I can remove the step about not being able to go to the payment step (and then i the step about complete will be click on button event), but I think it is important to show that we cannot go to the payment step.